src/components/layout/TheFooter.vue

```diff
@@ -112,12 +116,7 @@
           >
         </v-col>
       </v-row>
-      <v-bottom-navigation
-        class="bottom-navigation"
-        :elevation="1"
-        height="22"
-      >
-      </v-bottom-navigation>
+      <div class="bottom-navigation"></div>
```

Comment:
```suggestion
      <div class="bottom-decoration"></div>
```
Spontane Idee, weil es nur Optik ist und keine Navigationsfunktion hat.

Reply:
Done